keyboards/handwired/p60/keymaps/p3lim/keymap.c

```diff
@@ -0,0 +1,146 @@
+	* |-----------------------------------------------------------------------------------------+
+	* |       |     |        |                                         |        |     |         |
+	* `-----------------------------------------------------------------------------------------'
+	*/
+	[1] = LAYOUT(
+		C_ESC1,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______,
+		KC_CAPS, KC_HOME, KC_UP,   KC_END,  KC_PGUP, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_PSCR, _______, _______, _______, _______, _______, _______, _______,
+		_______, KC_VOLD, KC_MUTE, KC_VOLU, KC_F17,  _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______,                   _______,                            KC_NO,   _______, _______, _______
+	),
+
+	/* FN2
+	* ,-----------------------------------------------------------------------------------------.
+	* |     |     |     |     |     |     |     |     |     |     |     |     |     |           |
+	* |-----------------------------------------------------------------------------------------+
+	* |        |     |     |     |     |     |     |     |     |     |     |  Å  |     |        |
+	* |---------------------------------------------------------------------------------|       |
+	* |         |     |     |     |     |     |     |     |     |     |  Ø  |  Æ  |     |       |
+	* |-----------------------------------------------------------------------------------------+
+	* |           |     |     |     |     |     |     |     |     |     |     |           |     |
+	* |-----------------------------------------------------------------------------------------+
+	* |       |     |        |                                         |        |     |         |
+	* `-----------------------------------------------------------------------------------------'
+	*/
+	[2] = LAYOUT(
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, C_NO3,   _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, C_NO2,   C_NO1,   _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		KC_NO,   _______, _______,                   _______,                            _______, _______, _______, _______
+	),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record){
+	switch(keycode){
+		case C_ESC0: // layer 0
+			if(record->event.pressed){
+				if(get_mods() & MOD_MASK_SHIFT)
+					register_code(KC_GRAVE);
+				else
+					register_code(KC_ESCAPE);
+			} else {
+				if(get_mods() & MOD_MASK_SHIFT)
```

Comment:
Only issue with this, is if you release shift before this key, you may get a stuck key.  

May be better to unregister both, or use a static bool to track the status, and use that instead.

Reply:
What exact pattern are you concerned with? I've been using this for a long time and never had a key get stuck, and I'm unable to try and get it stuck intentionally.

Reply:
shift down, escape down, shift up, escape up.  

however, if you're fine with it, as is, then that's fine by me.

Reply:
Ah, guess I've never accidentally hit that combination.